src/Platform/Microsoft.Testing.Extensions.AzureDevOps/Microsoft.Testing.Extensions.AzureDevOps.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(MicrosoftTestingTargetFrameworks)</TargetFrameworks>
+    <RootNamespace>Microsoft.Testing.Extensions.Reporting</RootNamespace>
+  </PropertyGroup>
+
+  <!-- NuGet properties -->
```

Comment:
Do we want to ship it as stable? Or do we need to first wait until we are trying it ourselves for a while?